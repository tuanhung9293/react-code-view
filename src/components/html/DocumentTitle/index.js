import React from 'react'
import ReactDocumentTitle from 'react-document-title'

function DocumentTitle({title}) {
  return <ReactDocumentTitle title={
    title
      ? `${title} - Tuan`
      : 'Tuan Page'
  } />
}

export default DocumentTitle