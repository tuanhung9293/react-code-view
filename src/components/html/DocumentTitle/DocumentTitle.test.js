import React from 'react'
import {shallow} from 'enzyme'
import DocumentTitle from './'

describe('rendering', () => {
  it('renders DocumentTitle with props title', () => {
    expect(shallow(
      <DocumentTitle title="test title" />
    )).toMatchSnapshot()
  })
})