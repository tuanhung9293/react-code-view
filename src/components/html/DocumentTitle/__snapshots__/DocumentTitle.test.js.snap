// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`rendering renders DocumentTitle with props title 1`] = `
<SideEffect(DocumentTitle)
  title="test title"
/>
`;
